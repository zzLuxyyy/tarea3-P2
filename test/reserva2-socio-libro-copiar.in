crearSocio 12345678 Jim Dear 10 03 2004
crearFecha 01/01/1966
crearLibro 411222335 "Todos los fuegos el fuego" "Julio" "Cortázar" "Ocho cuentos que combinan lo cotidiano con lo fantástico" 1
crearReserva
socioReserva
libroReserva
copiarReserva
liberarReserva
Fin