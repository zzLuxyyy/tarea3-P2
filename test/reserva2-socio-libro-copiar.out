1>Socio de ci 12345678 creado exitosamente.
2>3>4>Reserva de libro de isbn 411222335 a socio 12345678 creada exitosamente.
5>Socio de la reserva:
Socio 12345678: Jim Dear
Fecha de alta: 10/3/2004
Géneros favoritos: 
6>Libro de la reserva:
Libro 411222335
Título: Todos los fuegos el fuego
Autor: Julio Cortázar
Descripción: Ocho cuentos que combinan lo cotidiano con lo fantástico
Género: 1
Fecha de edición: 1/1/1966
7>Reserva copiada. Datos de la copia:
Reserva de libro Todos los fuegos el fuego por Jim Dear.
8>Se ejecutó 'liberarTReserva' exitosamente.
9>Fin.
